era_5g_interface/task_handler_internal_q.py

```diff
@@ -18,18 +19,39 @@ class TaskHandlerInternalQ(TaskHandler):
     method externally.
     """
 
-    def __init__(self, sid: str, image_queue: Queue, **kw) -> None:
+    def __init__(self, sid: str, image_queue: Queue, if_queue_full="discard_oldest", queue_put_lock=None, **kw) -> None:
```

Comment:
if_queue_full - can you make it an enum or something other than a string?

Reply:
and please rename the image_queue to data_queue

Reply:
Please add a type annotation to `queue_put_lock`.

Reply:
done